control-plane/grpc/proto/v1/replica/replica.proto

```diff
@@ -157,6 +157,19 @@ message UnshareReplicaRequest {
   optional google.protobuf.StringValue pool_uuid = 5;
 }
 
+// Resize a replica to the requested size
+// For internal usage mostly.
+message ResizeReplicaRequest {
+  // id of the io-engine instance
+  string node_id = 1;
+  // uuid of the replica
+  google.protobuf.StringValue replica_id = 2;
+  // requested new size of the volume
+  uint64 requested_size = 3;
+    // id of the pool
```

Comment:
```suggestion
  // id of the pool
  string pool_id = 4;
  // uuid of the pool
  optional string pool_uuid = 5;
```

Reply:
There is no `pool_uuid` in the transport's ResizeReplica type as of today, hence didn't keep it here.

Reply:
ah I see we already have a mix of having uuid and not having it :( 
So seems like we can skip the pool altogether here as per existing requests